packages/express/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@loopback/express",
+  "version": "1.0.0",
+  "description": "@loopback/express",
+  "keywords": [
+    "loopback-extension",
+    "loopback"
+  ],
+  "main": "index.js",
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "build:watch": "lb-tsc --watch",
+    "pretest": "npm run clean && npm run build",
+    "test": "lb-mocha --allow-console-logs \"dist/__tests__\"",
+    "test:dev": "lb-mocha --allow-console-logs dist/__tests__/**/*.js",
+    "clean": "lb-clean dist *.tsbuildinfo .eslintcache"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/strongloop/loopback-next.git",
+    "directory": "packages/express"
+  },
+  "author": "IBM Corp.",
+  "license": "MIT",
+  "files": [
+    "README.md",
+    "index.js",
+    "index.d.ts",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "dependencies": {
+    "@loopback/context": "^3.5.1",
+    "@loopback/core": "^2.4.1",
```

Comment:
Please make sure this package is using the version of `@loopback/core` and `@loopback/context` as provided by the target LB4 application, so that app users are in full control.

Proposed changes:
- Move `@loopback/core` to peer dependencies
- Remove `@loopback/context` entirely, import context-related APIs from `@loopback/context`.

Reply:
We can remove `@loopback/context` and use `@loopback/core` only. But we need to apply the same rule for all other packages.

I'm also reluctant to use use `peerDependencies`. IMO, the only reliable way is to use the same set of versions released together and upgrade using `lb4 update`.

Let's defer this to a later time that we can visit all packages in `loopback-next`.